CMake/cdat_modules_extra/git_clone.sh.in

```diff
@@ -3,7 +3,7 @@
 cd "@CMAKE_INSTALL_PREFIX@"
 "@GIT_EXECUTABLE@" clone --no-checkout --depth 1 -b @BRANCH@ @GIT_URL@ "@GIT_TARGET@"
```

Comment:
I thought the problem would be here with -b option since that would not respect light weight tags.